spring/src/main/java/org/axonframework/spring/config/annotation/AnnotationCommandHandlerBeanPostProcessor.java

```diff
@@ -37,6 +37,7 @@
  * @author Allard Buijze
  * @since 0.5
  */
+@Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.